### PR TITLE
Setting up linters, Screencast of using gulp for task automation,  Setup Continous Integration with Travis 

### PR DESCRIPTION
**Pull request by rhenshaw56**
#### What does this PR do?

1) It displays the output of a task in the form of a link to a YouTube video showing auto-reloading of the browser while changes are being made to the html, css and js codes respectively.
2) It shows screenshots of Airbnb violations in the command line and hound CI comments on a PR which illustrates setting up of linters to watch for adherence to airbnb coding styles.
3) It also displays output of integrating Travis CI and a link to a passing build on my development branch

#### Description of Task to be completed?
1)A screen cast showing gulp watching and refreshing your browser when changes are made to your HTML, CSS & JS files.
2) Screenshots of Airbnb violations and HoundCI comments on a PR.
A screen 
3) [A link to a succesful travis build on development branch](https://travis-ci.org/andela-rhenshaw/inverted-index)
#### How should this be manually tested?

1)By watching the video on [YouTube](https://youtu.be/x9oTPYUQi60) 
2)By looking at the screenshots embedded here and in the output folder in the repo
3) By clicking on the link in the README.md file to checkout the Travis build

#### Any background context you want to provide?
1)Gulp is a toolkit for automating time-consuming tasks in your development workflow, so you can eliminate down-time and meet up your project deadlines. Check out [gulpjs](http://gulpjs.com/)

2)A linter is a tool that provides an API for flagging suspicious lines of written code when its syntax or logic has not met some predefined standard. The linting tool used to perform this task was eslint which is a pluggable and configurable linting tool for identifying and reporting on patterns in JavaScript.
3) [Travis CI](travis-ci.org) is a hosted distributed continuous integration service used to build and test software projects hosted at GitHub.

#### What are the relevant pivotal tracker stories?

N/A

#### Screenshots (if appropriate)
eslint-snapshot
houndci

#### Questions
N/A

### CODE REVIEW

**Comment by andela-cofor** [comment]
looks good to me...

**Comment by oreoluwade** [comment]
It looks to be in order as far as automation of tasks is concerned.

**Review by q-ode — COMMENTED** [review]
There are more changes in this PR than the description of the PR.

How many output submissions are you making?

**Comment by rhenshaw56** [comment]
@q-ode two submissions actually. Do i need to update the PR description or i should submit one at a time ?

**Comment by q-ode** [comment]
Update the PR please.

Also, I can't access the video.

**Comment by rhenshaw56** [comment]
@q-ode the work internet connection cant access the video, but it works on my personal connection.

**Comment by rhenshaw56** [comment]
@q-ode Updated PR description

File: 06-building-a-program-start-to-finish/automating-tasks/output/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#### OUTPUT
+- Link to Screencast on YouTube
+	-- [Browser watching and reloading with gulp](https://youtu.be/UK4qduIZT3k)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/automating-tasks/output/README.md]: Nice work!

I however don't see you implement the reload for modified JS files.

**Pull request closed without merging**